Tests/Tests/CustomEventSpec.swift

```diff
@@ -0,0 +1,50 @@
+import Quick
+import Nimble
+@testable import RInAppMessaging
+import CoreFoundation
+
+class CustomEventSpec: QuickSpec {
+    override func spec() {
+        describe("CustomEvent") {
+            let customAttributes =  [CustomAttribute(withKeyName: "Key1", withBoolValue: false),
+                                     CustomAttribute(withKeyName: "Key2", withDoubleValue: 3),
+                                     CustomAttribute(withKeyName: "Key3", withIntValue: 1)]
+            let customEvent = CustomEvent(withName: "Test4", withCustomAttributes: customAttributes)
+            context("CustomEvent.analyticsParameters") {
+                it("will return non nil dictionary value") {
```

Comment:
please add a case (or replace this one) when you expect a specific dictionary values from `customEvent.analyticsParameters`. (Hint: cast to `NSDictionary` to compare values)